feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/README.md

```diff
@@ -0,0 +1,107 @@
+# gNMI-1.12: Mixed OpenConfig/CLI Origin
+
+## Summary
+
+Ensure that both CLI and OC configuration can be pushed to the device at the
+same time.
+
+## Procedure
+
+Note: this test is intended to cover only the case of pushing some configuration
+along with OC paths - since it is unknown what CLI configuration would be
+required in the emergency case that is covered by this requirement. * Push
+non-overlapping mixed SetRequest specifying CLI for DUT port-1 and OpenConfig
+
+# for DUT port-2.
+
+along with OC paths, since it is unknown what CLI configuration would be
+required in the emergency case that is covered by this requirement.
+
```

Comment:
Looks like there is some cut/paste error here?  Can you correct?

Reply:
@hariramnat this commit is bad: https://github.com/openconfig/featureprofiles/pull/474/commits/dfa5c011b05a3287fff4e745bdd09392782da1ba

Reply:
Fixed it now.